modules/survey_accounts/help/survey_accounts.md

```diff
@@ -1,43 +1,20 @@
-# Survey Accounts
+### Survey Accounts Module
 
-## Purpose
+This module is used to create and manage survey forms that are sent to participants in a study.
 
-The survey accounts module is intended to manage the surveys which
-are sent to participants in a study. It is used to generate a survey
-key, and (optionally) automatically send it to the participant, if
-their email address is provided.
+Each survey form is generated with a unique URL. This survey link URL can be emailed directly to respondents for survey completion from home, or can be used in the clinic to capture data directly from participants, e.g. on a tablet in the waiting room during a study visit. 
 
-## Intended Users
+Use the Selection Filters to search for existing surveys by PSCID, Visit Label, Instrument, or the email of a participant who has already been sent a survey form. You can use any selection of filters needed. Click **Clear Filters** to clear the current selection of filters. Click **Show Data** to display the results in a data table. 
```

Comment:
2 quick comments on this para: 
* `You can use any selection of filters needed.` - can we remove? sounds a little clunky
* `Click **Show Data**` : There is no more Show Data button (This module was recently Reactified)

Reply:
Thanks! Sorry I forgot about the Show Data - removed both of those lines as requested.